hack/terraform-quickstart/start-cluster.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
There are several ways to define a dependency in TerraForm, but that's not like it needs to be done :) Just suggesting, maybe for later~